msteams-platform/sbs-graphactivity-feedbroadcast.yml

```diff
@@ -0,0 +1,381 @@
+
+        > [!NOTE]
+        > The **Scope name** should match with the **Application ID** URI with `/access_as_user` appended at the end.</br>
+           `api://bff2****.ngrok.io/00000000-0000-0000-0000-000000000000/access_as_user`
+
+        :::image type="content" source="~/assets/images/sbs-graphactivity-feedbroadcast/accessasuser.png" alt-text="Access as user" border="true":::
+
+    1. In the **Authorized client applications** section, identify the applications that you want to authorize for your app’s web application. 
+    
+    1. Select **Add a client application**. 
+
+          :::image type="content" source="~/assets/images/sbs-graphactivity-feedbroadcast/addaclient.png" alt-text="Add client" border="true":::
+
+    1. Enter **Client ID**: `1fec8e78-bce4-4aaf-ab1b-5451cc387264` for Teams mobile or desktop application. 
+
+          :::image type="content" source="~/assets/images/sbs-graphactivity-feedbroadcast/addclientapplication1.png" alt-text="Add client for Teams mobile or desktop application" border="true":::
+
+       Enter **Client ID**: `5e3ce6c0-2b1f-4285-8d4b-75ee78787346` for Teams web application.
+
+        :::image type="content" source="~/assets/images/sbs-graphactivity-feedbroadcast/addclientapplication2.png" alt-text="Add client application for Teams web application" border="true":::
+
+    1. Select **Authorized scopes**.
+
+         :::image type="content" source="~/assets/images/sbs-graphactivity-feedbroadcast/authorizedscope.png" alt-text="Authroized scope" border="true":::
+
+       The following image displays the client IDs:
+
+         :::image type="content" source="~/assets/images/sbs-graphactivity-feedbroadcast/clientapplications.png" alt-text="Client application" border="true":::
+
+    1. In the left panel, select **API Permissions**. 
+
+       > [!NOTE]
+       > Users need to consent to these permissions only if the Azure AD app is registered in a different tenant.
+
+    1. Select **Add a permission**.
+
+         :::image type="content" source="~/assets/images/sbs-graphactivity-feedbroadcast/configured-permission.png" alt-text="Add a permission" border="true":::
+
+    1. Select **Microsoft Graph**.
+
+    1. Select **Delegated permissions**.
+
+    1. Add the following permissions:</br>
+         * **User.Read** (enabled by default)
+         * **Directory.Read.All**
+         * **Directory.ReadWrite.All**
+         * **TeamsAppInstallation.ReadForUser**
+         * **TeamsAppInstallation.ReadWriteForUser**
+         * **TeamsActivity.Send**
+         * **TeamsAppInstallation.ReadForUser.All**
+
+          :::image type="content" source="~/assets/images/sbs-graphactivity-feedbroadcast/requestapplicationpermission.png" alt-text="Permission" border="true":::
+
+          :::image type="content" source=".~/assets/images/sbs-graphactivity-feedbroadcast/requestpermissions1.png" alt-text="Permission for request" border="true":::
+
+    1. Select **Add permissions**.
```

Comment:
Please add an image for the steps:

- Select Add permissions.
- Go to Add a permission.
- Select Add a permission.
- Select Microsoft Graph.
- Select Application permissions
-